.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           python-version: '3.x'
 
       - name: Install test dependencies.
-        run: pip3 install molecule docker yamllint ansible-lint
+        run: pip3 install molecule molecule-docker docker yamllint ansible-lint
```

Comment:
```suggestion
        run: pip3 install molecule[docker] docker yamllint ansible-lint
```

Reply:
Cool. Didn't know about this trick until now.